website/docs/d/data_lake_pipeline_run.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: data_lake_pipeline_run"
+sidebar_current: "docs-mongodbatlas-datasource-data-lake-pipeline-run"
+description: |-
+    Describes a Data Lake Pipeline Run.
+---
+
+# Data Source: mongodbatlas_data_lake_pipeline_run
+
+`mongodbatlas_data_lake_pipeline_run` describe a Data Lake Pipeline Run.
+
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```terraform
+data "mongodbatlas_data_lake_pipeline_run" "test" {
+  project_id = "PROJECT ID"
+  name = "DATA LAKE PIPELINE NAME"
```

Comment:
can we update this parameter to be pipeline_name to be consistent with Atlas Admin API docs if no concerns? https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Data-Lake-Pipelines/operation/getPipelineRun